### DISCUSSION TITLE
Sprinkles specificity

### DISCUSSION THREAD

**Discussion by p7gg** [vanilla-extract-css/vanilla-extract#1028] [Q&A]
Hi, how can I ensure sprinkles have higher specificity/priority than recipe, or any other api truly?
For example - see image -, I defined a `display: flex` via sprinkles but it won't override the `display: block` defined via recipes api.

![image](https://user-images.githubusercontent.com/69444613/223571748-57f1daaa-0a7b-4401-a18e-d1121e8a7ff9.png)

Link: [Codesanbox](https://codesandbox.io/p/github/AsphoUI/aspho/feat/loader?file=%2FREADME.md&workspace=%257B%2522activeFileId%2522%253A%2522cleytii9z0013g5cy9om8fgx8%2522%252C%2522openFiles%2522%253A%255B%2522%252Fpackages%252Fcore%252Fdev%252FApp.tsx%2522%255D%252C%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522gitSidebarPanel%2522%253A%2522COMMIT%2522%252C%2522spaces%2522%253A%257B%2522cleytkxxv0018356hcz7wt5yq%2522%253A%257B%2522key%2522%253A%2522cleytkxxv0018356hcz7wt5yq%2522%252C%2522name%2522%253A%2522Default%2522%252C%2522devtools%2522%253A%255B%257B%2522type%2522%253A%2522PREVIEW%2522%252C%2522taskId%2522%253A%2522dev%253Acore%2522%252C%2522port%2522%253A3000%252C%2522key%2522%253A%2522cleytkxxv0019356hgzb22meu%2522%252C%2522isMinimized%2522%253Afalse%257D%255D%257D%257D%252C%2522currentSpace%2522%253A%2522cleytkxxv0018356hcz7wt5yq%2522%252C%2522spacesOrder%2522%253A%255B%2522cleytkxxv0018356hcz7wt5yq%2522%255D%252C%2522hideCodeEditor%2522%253Afalse%257D)

**Comment by brandonpittman** [comment]
Is the display: block declared after the sprinkle in the CSS output?

**Reply by p7gg** [reply]
> Is the display: block declared after the sprinkle in the CSS output?

I'm not at my desk to confirm, but I'm pretty positive it is.

**Comment by chanwukim** [comment]
Is it solved?

**Comment by zhaohuanyuu** [comment]
I also encountered the problem that sprinkles have low priority and cannot take effect. Is there a better solution?

**Reply by chanwukim** [reply]
https://vanilla-extract.style/documentation/packages/sprinkles/#layer

There is a way to use `@layer` for global, reset, component, and sprinkles (utilities).
